docs/spelling_wordlist

```diff
@@ -855,6 +856,7 @@ wsgi
 www
 xe
 xgettext
+xreadlines
```

Comment:
Shouldn't be needed in the word list as long as all instances of this are surrounded by backticks.